### PR TITLE
Fix `Method Key` class, which had unnecessary attribute.

### PR DESCRIPTION
**Pull request by mskacelik**
fix:#2097
/cc @jmartisk 
I also added some __javadocs__.

### CODE REVIEW

File: client/implementation-vertx/src/main/java/io/smallrye/graphql/client/vertx/typesafe/VertxTypesafeGraphQLClientProxy.java [hunk 1]
```diff
@@ -305,6 +305,7 @@ private JsonObject request(MethodInvocation method) {
         if (clientModel == null) {
             query = queryCache.computeIfAbsent(method.getKey(), key -> new QueryBuilder(method).build());
         } else {
+            clientModel.getOperationMap().keySet().forEach(System.err::println);
```
Comment (jmartisk) [inline client/implementation-vertx/src/main/java/io/smallrye/graphql/client/vertx/typesafe/VertxTypesafeGraphQLClientProxy.java:308]: Please remove this line, or use a logger  on debug level

Comment (jmartisk) [reply client/implementation-vertx/src/main/java/io/smallrye/graphql/client/vertx/typesafe/VertxTypesafeGraphQLClientProxy.java:308]: Ah, this is logged for each request, so rather remove. This is something that only needs to be logged once, if at all

Comment (mskacelik) [reply client/implementation-vertx/src/main/java/io/smallrye/graphql/client/vertx/typesafe/VertxTypesafeGraphQLClientProxy.java:308]: Oh yeah, my bad. I left it while I was debugging.

**Review by jmartisk — APPROVED** [review]

**Comment by jmartisk** [comment]
Nice, thanks!

**Pull request merged**